src/App.js

```diff
@@ -1,19 +1,44 @@
 import React from 'react';
-import './App.css';
 
+import TodoList from './components/TodoList/TodoList';
+import NewTodo from './components/NewTodo/NewTodo';
 import users from './api/users';
+import todos from './api/todos';
+import './App.css';
+
+class App extends React.Component {
+  state = {
+    todos,
+  }
+
+  handleSubmitTodo = (newTodo) => {
```

Comment:
не совсем то, функция не сабмитит ничего, она добавляет новый элемент в существующий массив
```suggestion
  addNewTodo = (newTodo) => {
```